Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -390,6 +469,41 @@ public struct TLSConfiguration {
                                 applicationProtocols: applicationProtocols,
                                 shutdownTimeout: shutdownTimeout,
                                 keyLogCallback: keyLogCallback,
-                                renegotiationSupport: renegotiationSupport)
+                                renegotiationSupport: renegotiationSupport,
+                                additionalTrustRoots: [])
+    }
+
+    /// Creates a TLS configuration for use with client-side contexts.
+    ///
+    /// This provides sensible defaults, and can be used without customisation. For server-side
+    /// contexts, you should use `forServer` instead.
+    public static func forClient(cipherSuites: String = defaultCipherSuites,
+                                 verifySignatureAlgorithms: [SignatureAlgorithm]? = nil,
+                                 signingSignatureAlgorithms: [SignatureAlgorithm]? = nil,
+                                 minimumTLSVersion: TLSVersion = .tlsv1,
+                                 maximumTLSVersion: TLSVersion? = nil,
+                                 certificateVerification: CertificateVerification = .fullVerification,
+                                 trustRoots: NIOSSLTrustRoots = .default,
+                                 certificateChain: [NIOSSLCertificateSource] = [],
+                                 privateKey: NIOSSLPrivateKeySource? = nil,
+                                 applicationProtocols: [String] = [],
+                                 shutdownTimeout: TimeAmount = .seconds(5),
+                                 keyLogCallback: NIOSSLKeyLogCallback? = nil,
+                                 renegotiationSupport: NIORenegotiationSupport,
```

Comment:
This one can default `renegotiationSupport` (as we now have it in enough places) to `.none`.

Reply:
Addressed in https://github.com/apple/swift-nio-ssl/pull/284/commits/8e47d73fdc3270539a2dffdbcc09ef61120a840e.